### PR TITLE
bugfix: e2e Case Notes on EUA Queries

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: ui_automation_tests/features/end_user_advisory_queries.feature [hunk 1]
```diff
@@ -26,15 +26,14 @@ Feature: I want to raise an End User advisory enquiry to check if a particular e
...
-#   Then I see a notification on end user advisory list
+    Then I see a notification on end user advisory list
     When I open an end user advisory already created
     Then I see a notification for case note and can view the case note
     When I enter "This is my new case note" for case note
     And I click post note
     Then I can view "This is my new case note" in case notes
 
-  @LT_1474_ecju_queries @regression
+  @LT_1474_ecju_queries @regression @AT
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/end_user_advisory_queries.feature]: Need to remove this before pushing

Comment (andytaylorkainos) [reply ui_automation_tests/features/end_user_advisory_queries.feature]: damnit

File: ui_automation_tests/pages/end_user_advisory_page.py [hunk 2]
```diff
@@ -13,21 +13,13 @@ class EndUserAdvisoryPage(BasePage):
...
-        self.driver.set_timeout_to(10)
-        return return_value
-
-    def get_table_rows_and_position(self, _id: str):
-        elements = self.driver.find_elements_by_css_selector(self.TABLE_ROW)
-        no = helpers.get_element_index_by_text(elements, _id[:5] + "-" + _id[5:], complete_match=False)
-        return elements, no
+    def open_end_user_advisory(self, end_user_advisory_id):
+        end_user_advisory = self.driver.find_element_by_id(end_user_advisory_id)
+        end_user_advisory.find_elements_by_css_selector("a")[0].click()
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/pages/end_user_advisory_page.py]: I think this is risky. Could we instead give the <a> tag it's own ID like end_user_advisory_id+"-link" or something.

Comment (andytaylorkainos) [reply ui_automation_tests/pages/end_user_advisory_page.py]: good call!

**Pull request merged**